README.md

```diff
@@ -287,6 +287,7 @@ Additional lists you might find useful:
 *Plugins that implement object-relational mapping or data-mapping techniques.*
 
 - [ADmad/Sequence plugin](https://github.com/ADmad/cakephp-sequence) - Behavior for maintaining ordered list of records.
+- [Decimal plugin](https://github.com/dereuromark/cakephp-decimal) - A value object approach on handling decimals.
```

Comment:
- Invalid (plugin) name in overview, please see guidelines (you know them well 😉). 

```suggestion
- [CakeDecimal plugin](https://github.com/dereuromark/cakephp-decimal) - A value object approach on handling decimals.
```

Reply:
You are so right :)